services/movie_session.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from db.models import MovieSession
+from django.db.models import QuerySet
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    MovieSession.objects.create(show_time=movie_show_time,
+                                cinema_hall_id=cinema_hall_id,
+                                movie_id=movie_id)
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
```

Comment:
The function `get_movies_sessions` is expected to return a QuerySet, but the `.filter()` method in line 17 returns a list. Please correct the function return type to match the actual returned data type.